src/Common/src/Interop/Unix/libssl/Interop.libssl.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class libssl
+    {
+
+        [DllImport(Interop.Libraries.LibSsl, EntryPoint = "ERR_load_BIO_strings")]
```

Comment:
Yes.. unused ones should be removed. As an aside, calling from libssl works (holds true even for some *x509* and *BIO* methods)